content/en/network_monitoring/devices/netflow.md

```diff
@@ -0,0 +1,56 @@
+---
+title: NetFlow Monitoring
+kind: documentation
```

Comment:
```suggestion
kind: documentation
is_beta: true
```
We've been adding is_beta: true to our other docs. It no longer stops the page from being indexed. I think there might be some other functionality linked to it, like an icon or something, that the web team is going to fix for us.